subsys/logging/log_core.c

```diff
@@ -28,6 +28,161 @@ static bool initialized;
 
 static timestamp_get_t timestamp_func;
 
+/* MOJ KOD */
+
+#ifdef CONFIG_LOG_FRONTEND_PINS_IN_SEQUENCE
+	struct LOG_FRONTEND_PIN_NUMBERS log_frontend_pin_nums = {
```

Comment:
such array should by *const* to end up in flash instead of ram and *static*. However, this particular array is rather not needed since when gpios are in sequence it is not used.